stable/traefik/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: traefik
-version: 1.13.0
+version: 1.13.1
```

Comment:
Since this PR adds a feature, it's the minor version number that should be incremented.